tests/e2e/cucumber/environment/index.ts

```diff
@@ -138,12 +140,19 @@ AfterAll(() => state.browser && state.browser.close())
 setWorldConstructor(World)
 
 const cleanUpUser = async (adminUser: User) => {
+  let createdUser
+  if (config.keycloak) {
+    createdUser = KeycloakCreatedUser
+  } else {
+    createdUser = createdUserStore
+  }
   const requests: Promise<User>[] = []
-  createdUserStore.forEach((user) => {
+  createdUser.forEach((user) => {
     requests.push(api.provision.deleteUser({ user, admin: adminUser }))
```

Comment:
little comments can be useful for future 
add comment about todo also

Reply:
I have changed by calling `KeycloakCreatedUser` while only deleting `user` from keyclock.
and i donot make change on it